ConsoleCalculator/Calculator.cs

```diff
@@ -3,12 +3,228 @@
 namespace ConsoleCalculator
 {
     public class Calculator
-    {
-
-        public string SendKeyPress(char key)
+    {
+        static string number = "";
+        static string operation = "";
+        static string secondNumber = "";
+
+        public string SendKeyPress(char key)
+        {
+            switch(key)
+            {
+                case 'c':
+                case 'C':
+                    ResetAll();
+                    return "0";
+                case '.':
+                case 'S':
+                case 's':
+                case '0':
+                case '1':
+                case '2':
+                case '3':
+                case '4':
+                case '5':
+                case '6':
+                case '7':
+                case '8':
+                case '9':
+                    return ShowNumber(key);
+                
+                case '+':
+                    SetCalculationSymbol(key);
+                    break;
+                case '-':
+                    SetCalculationSymbol(key);
+                    break;
+                case 'x':
+                    SetCalculationSymbol(key);
+                    break;
+                case '/':
+                    SetCalculationSymbol(key);
+                    break;
+                case '=':
+                    if (operation != "")
+                    {
+                        string previousOperation = operation;
+                        operation = "";
+                        return DoCalculation(previousOperation);
+                    }
+                    break;
+                default:
+                    return "";
+                
+            }
+
+            return number;
+        }
+
+        private string DoCalculation(string operation)
+        {
+            string answer="";
+            if(operation == "+")

```

Comment:
There is excessive use of conditional statements. Can you Create separate classes for different operations which have a common Interface having a common method name? Each class will perform there own implementation.